shipstation_integration/shipments.py

```diff
@@ -158,3 +156,35 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+def create_shipment(shipment, delivery_note, store):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
+	shipment_doc.description_of_content = "Farm plastic"
+	shipment_doc.value_of_goods = delivery_note.total
```

Comment:
Is this the net total of all the delivery note items?

Reply:
Extra context: I think the shipment automatically calculates this value, so maybe we just let Frappe handle it

Reply:
Yes